util/genbuild_h/genbuild_h.sh

```diff
@@ -66,6 +66,10 @@ printf "#define COREBOOT_VERSION %s\n" "\"$KERNELVERSION\""
 
 #See if the build is running in a git repo and the git command is available
 printf "/* timesource: $TIMESOURCE */\n"
+
+printf "#define DASHARO_VERSION \"%s\"\n" `git describe --tags --match "asus_kgpe-d16_v[0-9]*.[0-9]*.[0-9]*" | sed 's/asus\_kgpe\-d16\_//' | sed 's/\(v[0-9]\.[0-9]\.[0-9]\+\).*/\1/'`
+printf "#define DASHARO_MAJOR_VERSION %d\n#define DASHARO_MINOR_VERSION %d\n#define DASHARO_PATCH_VERSION %d\n" `git describe --tags --match "asus_kgpe-d16_v[0-9]*.[0-9]*.[0-9]*" | sed 's/asus\_kgpe\-d16\_//' | sed 's/v\([0-9]\)\.\([0-9]\)\.\([0-9]\+\).*/\1 \2 \3/'`
```

Comment:
I assume this is still problematic, it should not be paltform specific?

Reply:
Right, it's still a problem

Reply:
Changed to get DASHARO_VERSION and others from CONFIG_LOCALVERSION. If that's not set, DASHARO_VERSION will be set to `*INVALID*`